src/outbox/outboxActions.js

```diff
@@ -176,7 +181,9 @@ export const addToOutbox = (narrow: Narrow, content: string) => async (
       avatar_url: userDetail.avatar_url,
       isOutbox: true,
       reactions: [],
+      deferSending,
     }),
   );
   dispatch(sendOutbox());
```

Comment:
This line can be skipped when `deferSending`, and that might make it a bit easier to think about what's intended to happen here.